orbs/circleci/circle-wait-job

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env ruby
+
+require("optparse")
+require("net/http")
+require("json")
+require("date")
+
+class CLI
+  attr_reader(:token, :organization, :project, :job, :current_job, :branch, :interval, :vcs, :build_number)
+
+  def initialize
+    @branch = ENV["CIRCLE_BRANCH"]
+    @interval = "60"
+    @vcs = "github"
+    @build_number = ENV["CIRCLE_BUILD_NUM"]
+    @token = ENV["CIRCLE_TOKEN"]
+    @current_job = ENV["CIRCLE_JOB"]
+    @job = ENV["CIRCLE_JOB"]
```

Comment:
**Question**
What is the difference between current job and job?

Reply:
Since the concurrent management is done in a dedicated job, `current_job` is the name of that concurrent management job (ex: https://github.com/jobteaser/ui-jobteaser/pull/124/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R247).
On the other hand, `job` is the name of the job for which you want to ensure the unicity (in `ui-jobteaser` this is the deploy job called `deploy_prod`, see https://github.com/jobteaser/ui-jobteaser/pull/124/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R255).

Reply:
Ok, understood now!
Maybe it's not a "job" but a "step" no?

Reply:
The handling for concurrent management is really a new and dedicated job (see https://github.com/jobteaser/circleci/pull/48/files#diff-1d58629fb3085b74cbe65b38f786be5fda21f7f70da14e085841f51479c117f9R48 for the definition and https://github.com/jobteaser/ui-jobteaser/pull/124/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R243 for the usage). And this new job is made up of several steps (see https://github.com/jobteaser/circleci/pull/48/files#diff-1d58629fb3085b74cbe65b38f786be5fda21f7f70da14e085841f51479c117f9R66). Or maybe we're not talking about the same thing 😅  ?